src/app/pages/wallets/wallet-details/transaction-editor/pages-wallet-transaction-editor.component.spec.ts

```diff
@@ -90,28 +90,42 @@ describe('TransactionEditorComponent', () => {
 
     describe('amount field validation', () => {
       describe('error', () => {
-        it('should show validation error if value has invalid format', () => {
-          component.isLoadingTransactionCategories = false;
-          component.form.get('amount')?.setValue(12.3333);
-          component.form.get('amount')?.markAsTouched();
-
-          fixture.detectChanges();
-          const errorMessageDiv = fixture.debugElement.query(By.css('.mat-error'));
-          const errorMessageText: HTMLDivElement = errorMessageDiv.nativeElement as HTMLDivElement;
-
-          expect(errorMessageText.textContent!.trim()).toBe(ERROR_MESSAGE_AMOUNT_INVALID_FORMAT);
+        describe('should show validation error if value has invalid format', () => {
+          it('12.3333', () => {
+            component.isLoadingTransactionCategories = false;
+            component.form.get('amount')?.setValue(12.3333);
+            component.form.get('amount')?.markAsTouched();
+
+            fixture.detectChanges();
+            const errorMessageDiv = fixture.debugElement.query(By.css('.mat-error'));
+            const errorMessageText: HTMLDivElement = errorMessageDiv.nativeElement as HTMLDivElement;
+
+            expect(errorMessageText.textContent!.trim()).toBe(ERROR_MESSAGE_AMOUNT_INVALID_FORMAT);
+          });
         });
       });
 
       describe('success', () => {
-        it('should show no validation error if amount has valid format', () => {
-          component.form.get('amount')?.setValue(10.21);
-          component.form.get('amount')?.markAsTouched();
+        describe('should show no validation error if amount has valid format', () => {
+          it('0', () => {
```

Comment:
```suggestion
        describe('validation errors', () => {
          it('should not show validation error if amount is equal to 0', () => {
```
Other test descriptions should be changed in similar way. `it` should describe what `should` happen. `describe` is just grouping a bunch of related tests together and label them.